analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MultilineBlocksWithoutBrace.old.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
```

Comment:
Please add your new test cases to the end of the original file. This `old` file doesn't make sense in long term.

Reply:
@pavel-mikula-sonarsource I want to remove that one, but first wanted to show, that I did not break anything.